src/js/tabs/pid_tuning.js

```diff
@@ -653,6 +662,8 @@ TABS.pid_tuning.initialize = function (callback) {
             ADVANCED_TUNING.dMinGain = parseInt($('.dminGroup input[name="dMinGain"]').val());
             ADVANCED_TUNING.dMinAdvance = parseInt($('.dminGroup input[name="dMinAdvance"]').val());
 
+            ADVANCED_TUNING.useIntegratedYaw = $('input[id="useIntegratedYaw"]').is(':checked') ? 1 : 0;
+
```

Comment:
Newline.

Reply:
I like to have an empty like before and after each element of the screen, but no problem removing it. What line must I remove? Before, after or both?

Reply:
Na, there's no hard rules about that, just leave it if you like it.

Reply:
Removed the last line. You have reason, in this situation is better without it ;)